### PR TITLE
Translate library/pathlib part 2/2

### PR DESCRIPTION
**Pull request by mindihx**
issue: https://github.com/python/python-docs-zh-tw/issues/665
html: 
[pathlib.zip](https://github.com/python/python-docs-zh-tw/files/13852284/pathlib.zip)

### CODE REVIEW

File: library/pathlib.po [hunk 1]
```diff
@@ -595,51 +595,63 @@ msgstr ""
...
+"對路徑物件做系統呼叫。有三種方式可以實例化實體路徑："
 
 #: ../../library/pathlib.rst:735
 msgid ""
 "A subclass of :class:`PurePath`, this class represents concrete paths of the "
 "system's path flavour (instantiating it creates either a :class:`PosixPath` "
 "or a :class:`WindowsPath`)::"
 msgstr ""
+":class:`PurePath` 的子類別，此類別表示系統的路徑類型的實體路徑（實例化時會建"
+"立一個 :class:`PosixPath` 或 :class:`WindowsPath`）： ::"
```
Comment (cschan1828) [inline library/pathlib.po:616]:
```suggestion
"立一個 :class:`PosixPath` 或 :class:`WindowsPath`）::"
```

**Comment by cschan1828** [comment]
Sorry for the delayed response. I've been quite busy recently and haven't had the chance to review this PR yet.

File: library/pathlib.po [hunk 2]
```diff
@@ -743,12 +777,16 @@ msgid ""
 "typically, case-sensitive on POSIX, and case-insensitive on Windows. Set "
 "*case_sensitive* to ``True`` or ``False`` to override this behaviour."
 msgstr ""
+"預設情況下，或者當 *case_sensitive* 關鍵字參數被設定為 ``None`` 的時候，此方"
```
Comment (mattwang44) [inline library/pathlib.po]:
```suggestion
"預設情況下，或者當 *case_sensitive* 僅限關鍵字引數被設定為 ``None`` 的時候，此方"
```

File: library/pathlib.po [hunk 3]
```diff
@@ -743,12 +777,16 @@ msgid ""
...
+"法會使用平台特定的大小寫規則來比對路徑；通常在 POSIX 上會區分大小寫，而在 "
+"Windows 上不區分大小寫。將 *case_sensitive* 設成 ``True`` 或 ``False`` 會覆寫"
+"這個行為。"
 
 #: ../../library/pathlib.rst:936
 msgid ""
 "Using the \"``**``\" pattern in large directory trees may consume an "
 "inordinate amount of time."
-msgstr ""
+msgstr "在很大的目錄樹裡使用 “``**``” 可能會耗費過多的時間。"
```
Comment (mattwang44) [inline library/pathlib.po]:
```suggestion
msgstr "在很大的目錄樹裡使用 \"``**``\" 可能會耗費過多的時間。"
```

File: library/pathlib.po [hunk 4]
```diff
@@ -783,18 +827,24 @@ msgid ""
...
 msgstr ""
+"如果該路徑指向一個普通檔案（或者是一個指向普通檔案的符號連結）則回傳 "
+"``True``，如果指向其他類型的檔案則回傳 ``False``。"
 
 #: ../../library/pathlib.rst:975
 msgid ""
 "Return ``True`` if the path points to a junction, and ``False`` for any "
 "other type of file. Currently only Windows supports junctions."
 msgstr ""
+"如果該路徑指向一個 junction 則回傳 ``True``，對其他類型的檔案則回傳 "
```
Comment (mattwang44) [inline library/pathlib.po]: hmmm junction 該翻譯嗎？

File: library/pathlib.po [hunk 5]
```diff
@@ -861,19 +926,24 @@ msgid ""
 "directory after creating the iterator, whether a path object for that file "
 "be included is unspecified."
 msgstr ""
+"子路徑會以任意順序被 yield，且不會包含特殊項目 ``’.’`` 和 ``’..’``。如果一個"
```
Comment (mattwang44) [inline library/pathlib.po]:
```suggestion
"子路徑會以任意順序被 yield，且不會包含特殊項目 ``'.'`` 和 ``'..'``。如果一個"
```

File: library/pathlib.po [hunk 6]
```diff
@@ -861,19 +926,24 @@ msgid ""
...
+"檔案在建立這個迭代器之後加到該目錄或從目錄刪除，是否會包含這個檔案的路徑物件"
```
Comment (mattwang44) [inline library/pathlib.po]:
```suggestion
"檔案在建立這個疊代器之後加到該目錄或從目錄刪除，是否會包含這個檔案的路徑物件"
```

File: library/pathlib.po [hunk 7]
```diff
@@ -861,19 +926,24 @@ msgid ""
...
-msgstr ""
+msgstr "透過由上而下或由下而上地走訪目錄樹產生目錄樹裡的檔案名稱。"
 
 #: ../../library/pathlib.rst:1068
 msgid ""
 "For each directory in the directory tree rooted at *self* (including *self* "
 "but excluding '.' and '..'), the method yields a 3-tuple of ``(dirpath, "
 "dirnames, filenames)``."
 msgstr ""
+"對每個以 *self* 為根且在目錄樹裡的目錄（包含 *self* 但不包含 ‘.’ 和 ‘..’ ），"
```
Comment (mattwang44) [inline library/pathlib.po]:
```suggestion
"對每個以 *self* 為根且在目錄樹裡的目錄（包含 *self* 但不包含 '.' 和 '..' ），"
```

File: library/pathlib.po (hunk 4 above)
Comment (mindihx) [reply library/pathlib.po]: 翻成連接點可以嗎？
https://learn.microsoft.com/zh-tw/sysinternals/downloads/junction

**Review by mattwang44 — APPROVED** [review]

**Pull request merged**